bentoml/cli/bento_management.py

```diff
@@ -175,39 +172,80 @@ def list_bentos(
         _print_bentos_info(result, output)
 
     @cli.command(
-        help='Delete bento. To delete multiple bentos provide the name '
-        'version tag separated by "," for example "bentoml delete name:v1,name:v2',
+        help='Delete bentos. To delete a bento service use "--bento-name" and '
+        '"--bento-version" option: "bentoml delete --bento-name IrisClassifier '
+        '--bento-version 0.1.0". To delete multiple bentos, use "--bento-name" and/or '
+        '"--labels" to filter. To delete all bento services use "--all" option'
+    )
+    @click.option(
+        '--all', is_flag=True, help='Use this flag to remove all BentoServices'
+    )
+    @click.option(
+        '--tag',
```

Comment:
this will still be a breaking change right? since it used to be an argument?

Reply:
Yes, this would be a breaking change.

Reply:
@yubozhao can you make it non-breaking and support the old format?